test/mock/rtps/EDP/fastdds/rtps/builtin/discovery/endpoint/EDP.h

```diff
@@ -34,21 +34,25 @@ namespace rtps {
 
```

Comment:
As `EDP.h` was not included finally in the PR, I think this file should not be included nor modified by this PR.